### PR TITLE
Fix FOUC in privacy policy message

### PR DESCRIPTION
**Pull request by jonnywyatt**
Remove privacy policy styles from main stylesheet as already in critical; increase specificity of privacy policy styles

 🐿 v2.8.0

### CODE REVIEW

**Comment by liamkeaton** [comment]
Am I going mad or did you do this yesterday?

File: main.scss [hunk 1]
```diff
@@ -36,4 +36,3 @@ $o-tooltip-is-silent : false;
 @import './src/components/desktop-app-banner/main';
 @import './src/components/anon-subscribe-now-teal/main';
 @import './src/components/cookie-consent/main';
-@import './src/components/privacy-policy/main';
```
Comment (remybach) [inline main.scss]: Oops... hold on... are you meant to be removing this?

**Comment by jonnywyatt** [comment]
@liamkeaton ha ha you're not going mad. I did remove from main.scss. However the specificity issue meant the styles broke so I had to revert it.
Raising the specificity is a sticking plaster, those styles need to move in to n-alert-banner then it won't be needed. I'll make a tech debt note on our board

File: main.scss (hunk 1 above)
Comment (jonnywyatt) [reply main.scss]: Yes. The issue is, they're also included in critical.scss. When the CSS is built, it de-dupes in favour of the last occurrence of a style. As that second occurrence was in main.scss, that's the one that remains. That stylesheet is lazily loaded, but the markup for the message is server-rendered, hence the FOUC

**Comment by liamkeaton** [comment]
@jonnywyatt awesome, I believe we will be moving to `o-message` soon which may make things easier/different...

**Pull request merged**